### PR TITLE
Fixed #26834 -- Made IntegerField forward min/max validators to form …

### PR DESCRIPTION
**Pull request by sgzsh269**
…field

The change forwards the user set limit values of MinValueValidator/MaxValueValidator
defined in the model IntegerField to the form field using the formfield() method.

Thanks Markus Holtermann for help with the patch.

### CODE REVIEW

**Comment by timgraham** [comment]
buildbot, add to whitelist.

File: tests/model_fields/test_integerfield.py [hunk 1]
```diff
@@ -132,6 +132,20 @@ def test_coercing(self):
         instance = self.model.objects.get(value='10')
         self.assertEqual(instance.value, 10)
 
+    def test_min_max_validator_limit_value_passed_to_formfield(self):
+        """
+        IntegerField passes the limit value set in MinValueValidator and
+        MaxValueValidator to the form fields using formfield() method.
+        """
+        if1 = models.IntegerField()
+        if2 = models.IntegerField(validators=[validators.MinValueValidator(10),
```
Comment (MarkusH) [inline tests/model_fields/test_integerfield.py]: Could you wrap this as
```python
if2 = models.IntegerField(
    validators=[validators.MinValueValidator(10), validators.MaxValueValidator(100)]
)
```
please.

**Comment by MarkusH** [comment]
What happens when a field has more than one `MaxValueValidator` (or `MinValueValidator`) where the boundary values differ, e.g.
```python
field = IntegerField(validators=[MaxValueValidator(5), MaxValueValidator(10)])
```

**Comment by timgraham** [comment]
I think when we talked at the sprints, you indicated there was another issue that needed to be fixed before this one. Do you have any follow up with details about that? Should we close this in the meantime?

**Comment by sgzsh269** [comment]
@timgraham yes, the issue is related to the validators property, it needs further evaluation. Haven't gotten a chance to work on it but plan to do it soon. 
I think @MarkusH mentioned at the time that the fix for the validators property issue could be included in this same PR.
Should I create a separate ticket and subsequently a separate PR for the validators property issue?

**Comment by timgraham** [comment]
A separate ticket/PR is fine. Sometimes a PR will include multiple commits but separating them if they stand on their own is fine too.

**Comment by timgraham** [comment]
Closing since this isn't suitable for merge yet.

**Comment by mluszczyk** [comment]
@sgzsh269 are you still working on this? I'd like to see this feature included into Django and I'll be happy to help with this patch. Could you elaborate on the "issue related to the validators property", is there a ticket for it?

**Pull request closed without merging**